qdrant/collections.go

```diff
@@ -59,9 +60,14 @@ func (c *Client) UpdateCollection(ctx context.Context, request *UpdateCollection
 
 // Drop collection and all associated data.
 func (c *Client) DeleteCollection(ctx context.Context, collectionName string) error {
-	_, err := c.GetCollectionsClient().Delete(ctx, &DeleteCollection{
+	res, err := c.GetCollectionsClient().Delete(ctx, &DeleteCollection{
 		CollectionName: collectionName,
 	})
+
+	if !res.GetResult() {
+		return newQdrantErr(errors.New("failed to delete collection"), "DeleteCollection", collectionName)
+	}
+
 	if err != nil {
 		return newQdrantErr(err, "DeleteCollection", collectionName)
 	}
```

Comment:
Please switch order, first err, after that assume res is an instance. (empty lines are not recommended in Go style guide)

Reply:
Done.
[`24ae329` (#51)](https://github.com/qdrant/go-client/pull/51/commits/24ae3299d4145aa2d3f25aaacdf68fa7caf69aca)

Reply:
Removing the lines.